api/organisations/views.py

```diff
@@ -221,7 +221,7 @@ def get_hosted_page_url_for_subscription_upgrade(self, request, pk):
 
     @swagger_auto_schema(
         deprecated=True,
-        operation_description="Please use ​​/api​/v1​/organisations​/{organisation_pk}​/usage-data​/",
+        operation_description="Please use /api/v1/organisations/{organisation_pk}/usage-data/",
```

Comment:
What are these changes?

Reply:
Ah yes, some sneaky secret characters that I wanted to get rid of so snuck into this PR. 

This is the diff in my IDE

<img width="1182" alt="image" src="https://github.com/user-attachments/assets/8a058fee-0143-45c4-bce3-bc16f604f1ed">